mg/cmd.go

```diff
@@ -57,6 +59,8 @@ func (w *CmdOutputWriter) write(writeIfClosed bool, p []byte) (int, error) {
 	return len(p), nil
 }
 
+// Close writes provided output(s) and closes the writer. It returns
+// os.ErrClosed if Close is already been called.
```

Comment:
s/`is`/`has`

Reply:
maybe also mention that if `w.Closer` isn't nil, `w.Closer.Close()` will be called